roles/mesos/tasks/RedHat.yml

```diff
@@ -11,16 +11,18 @@
 # match those of the roles.
 
 - name: Check Mesos Master running
-  shell: sudo status mesos-master
```

Comment:
The `Debian.yml` file contains the following. Maybe we can move these lines to `main.yml`?

```yaml
# The Mesos package installs both the master and agent. Make the Mesos services
# match those of the roles.
- name: Disable Mesos Master for non-master hosts
  service: name=mesos-master state=stopped enabled=no
  when: "'mesos-master' not in group_names"

- name: Disable Mesos Agent for non-agent hosts
  service: name=mesos-slave state=stopped enabled=no
  when: "'mesos-agent' not in group_names"
```